### PR TITLE
added change in taxi serivce

### PR DESCRIPTION
**Pull request by Anish22prakash**
added taxi driver method, register , get taxi driver and get all taxi driver.

### CODE REVIEW

File: Taxi_Booking_Management/Services/TaxiDriver/TaxiDriverService.cs [hunk 1]
```diff
@@ -1,6 +1,110 @@
...
+                    _loggerManger.LogInfo("taxi driver mobile is already exist");
+                    return message;
+                }
+                await _context.drivers.AddAsync(taxiDriver);
+                await _context.SaveChangesAsync();
+                message = MessagesAlerts.SuccessfullSave;
+                _loggerManger.LogInfo($"taxi driver is successfully register with name {taxiDriver.DriverName}");
+                return message;
+            }
+            catch (Exception ex)
```
Comment (Ash1998-05-19) [inline Taxi_Booking_Management/Services/TaxiDriver/TaxiDriverService.cs:103]: please use error handling properly

**Review by Ash1998-05-19 — CHANGES_REQUESTED** [review]
Please resolve comment

**Review by Ash1998-05-19 — APPROVED** [review]

File: Taxi_Booking_Management/Services/TaxiDriver/TaxiDriverService.cs (hunk 1 above)
Comment (Anish22prakash) [reply Taxi_Booking_Management/Services/TaxiDriver/TaxiDriverService.cs:103]: changes done

**Review by Anish22prakash — COMMENTED** [review]
added driver and owner service method

**Review by Ash1998-05-19 — COMMENTED** [review]
Reviewed

**Review by Ash1998-05-19 — APPROVED** [review]
reviewed all changes

**Pull request merged**